Common/tests/ListSessionsRequestValidatorTest.cs

```diff
@@ -134,4 +134,12 @@ public void ListSessionsRequestNegativePageSizeShouldFail()
     Assert.IsFalse(validator_.Validate(validListSessionsRequest_)
                              .IsValid);
   }
+
+  [Test]
+  public void ListSessionsZeroNegativePageSizeShouldFail()
+  {
+    validListSessionsRequest_!.PageSize = 0;
```

Comment:
I think you want a negative number here ^^

Reply:
The test has the wrong name^^ I have 2 tests a negative and a 0